Expensify/App
# Add some hitSlop around the send button

### Details
Adds some extra touchability margins around the "send chat" button.

### Fixed Issues
$ https://github.com/Expensify/App/issues/6662

### Tests / QA Steps
1. Type a message.
1. Touch just outside the send button (but still within the bounds of the composer)
1. Verify that the message sends.

### Tested On

- [] Web

### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
